cmd/cli/trafficpolicy_check.go

```diff
@@ -33,17 +36,20 @@ osm policy check-pods bookbuyer-client bookstore-server
 `
 
 const (
-	namespaceSeparator = "/"
-	osmConfigMapName   = "osm-config"
-	serviceAccountKind = "ServiceAccount"
+	namespaceSeparator       = "/"
+	defaultOsmMeshConfigName = "osm-mesh-config"
+	serviceAccountKind       = "ServiceAccount"
 )
 
 type trafficPolicyCheckCmd struct {
-	out             io.Writer
-	sourcePod       string
-	destinationPod  string
-	clientSet       kubernetes.Interface
-	smiAccessClient smiAccessClient.Interface
+	out               io.Writer
+	sourcePod         string
+	destinationPod    string
+	clientSet         kubernetes.Interface
+	smiAccessClient   smiAccessClient.Interface
+	configClient      osmConfigClient.Interface
+	restConfig        *rest.Config
+	osmMeshConfigName string
```

Comment:
This shouldn't be necessary if `--osm-namespace` is passed. Could you rely on that instead?

Reply:
Let's keep the CLI behavior as before, and make such changes in a separate PR. So for this PR, I suggest to rely on the `--osm-namespace` flag because there is 1:1 mapping between a mesh and OSM's namespace.

Reply:
oh actually, the meshConfigName argument is optional. It falls back to `osm-mesh-config` if not provided. From the CLI perspective, the behavior is not changed.

I will update the trafficpolicy_check test cases to reflect it.

Reply:
Can we hardcode the mesh-config name and update this CLI in a separate PR? Currently, the cli code hardcodes the ConfigMap name as `osm-config`.

Reply:
Sure